src/components/MyAccount/index.tsx

```diff
@@ -246,25 +218,46 @@ export default function MyAccount({ user, organizations, edit, reloadUser, reloa
           />
         </>
       )}
-      <TfMain>
-        <Grid container spacing={3} className={edit ? classes.editContainer : ''}>
-          <Grid item xs={12} className={classes.titleContainer}>
-            <Grid item xs={7}>
-              <h1 className={classes.title}>{strings.MY_ACCOUNT}</h1>
-              <p>{strings.MY_ACCOUNT_DESC}</p>
-            </Grid>
-            <Grid item xs={5} className={classes.centered}>
-              <Button
-                id='edit-account'
-                label={isMobile ? strings.EDIT : dictionary.EDIT_ACCOUNT}
-                onClick={() => history.push(APP_PATHS.MY_ACCOUNT_EDIT)}
-                size='medium'
-                priority='secondary'
-              />
-            </Grid>
-          </Grid>
+      <PageHeaderWrapper nextElement={contentRef.current}>
+        <Box
+          display='flex'
+          justifyContent='space-between'
+          marginBottom={theme.spacing(4)}
+          paddingLeft={theme.spacing(3)}
+        >
+          <Box>
+            <Typography marginBottom={theme.spacing(1.5)} fontSize='24px' fontWeight={600}>
+              {strings.MY_ACCOUNT}
+            </Typography>
+            <Typography fontSize='14px' fontWeight={400}>
+              {strings.MY_ACCOUNT_DESC}
+            </Typography>
+          </Box>
+          <Button
+            id='edit-account'
+            icon='iconEdit'
+            label={isMobile ? strings.EDIT : dictionary.EDIT_ACCOUNT}
```

Comment:
I think for mobile we are not showing any label, just the icon. But maybe @chudo76 can confirm that

Reply:
I think you are correct. I'll make that change now and we can always change it back if needed.